### ISSUE TITLE
[Enhancement] - Add to find inside a note the feature to search backwards.

### ISSUE THREAD

**Issue by joaocarvalhoopen** [Zettlr/Zettlr#3358]
### Description

When you make a fast search inside a note file with CTRL+F you should appear a icon it the small search windows to let you choose if you want to find the next or the previous term from the current cursor position that is updated with each search/find.

It  should be added a new key binding to find the previous term. 
I don’t know what the normal search backwards keyboard binding should be for each operating system, please suggest.

### Proposed Changes

-Add the functionality to search backward from the current text position. 
-Add an icon for previous word and other for next word.
-Add the new key binding for previous position.

### Caveats

No Caveats and I think this is really important feature.

### Do you Wish to Attempt Implementing this Yourself?

No

### Zettlr Version

Stable (most recent version)

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Operating System Version

Linux Ubuntu 20.04

**Comment by kyaso** [comment]
There is already a key binding for search backwards within a file: `Cmd/Ctrl + Shift + F.

But I agree that having clickable buttons for searching forward/backwards would be a nice addition.

**Comment by joaocarvalhoopen** [comment]
Thank you @kyaso  I put here the reference to the source file:

```/source/common/modules/markdown-editor/plugins/search.ts ```


```function searchPrevious()```

**Comment by nathanlesage** [comment]
Agree, the find-in-file function can really use an overhaul to make it nicer, and such a button is a good idea.
